ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -679,3 +679,11 @@ ENABLE_PASSWORD_HASHING = True
 {% if localsettings.get('ENABLE_USED_PASSWORDS_CHECK') %}
 ENABLE_USED_PASSWORDS_CHECK = True
 {% endif %}
+
+{% if localsettings.get('ENIKSHAY_PRIVATE_API_USERS') %}
+ENIKSHAY_PRIVATE_API_USERS = True
```

Comment:
this should probably be `ENIKSHAY_PRIVATE_API_USERS = localsettings.get('ENIKSHAY_PRIVATE_API_USERS')` unless i'm misunderstanding (same with the below)